B827EBFFFEC01A9E.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEC01A9E",
+    "servers": [
+      {
+        "server_address": "asia-se.thethings.network",
```

Comment:
Hi @binyangvip, is the address really correct? I don't see it as a valid router on https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway

Reply:
thanks,this is my wrong.now,I've updated it.